webapp/views/wpt-results.js

```diff
@@ -381,7 +386,7 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
         type: Array,
         computed: 'computeDisplayedTests(path, searchResults)',
       },
-      metadata: Object,
+      metadataMap: Object,
```

Comment:
I have a bit concern for a race condition here, when metadataMap is fetched after the table cell is populated and `hasMetadata` will return false. But it has never happened in my experiment.

Reply:
(The function in question is actually `shouldDisplayMetadata`.)

I think there *is* a race condition here. One way to fix it is to add `metadataMap` as a parameter to `shouldDisplayMetadata`, so that `shouldDisplayMetadata` will be re-evaluated every time `metadataMap` is  updated.